packages/react-components/react-progress/src/stories/Progress/ProgressDefault.stories.tsx

```diff
@@ -1,4 +1,28 @@
 import * as React from 'react';
-import { Progress, ProgressProps } from '@fluentui/react-progress';
+import { Progress } from '@fluentui/react-progress';
 
-export const Default = (props: Partial<ProgressProps>) => <Progress {...props} />;
+const intervalDelay = 100;
+const intervalIncrement = 0.01;
+
+export const Default = () => {
+  const [percentComplete, setPercentComplete] = React.useState(0);
+
+  React.useEffect(() => {
+    const id = setInterval(() => {
+      setPercentComplete((intervalIncrement + percentComplete) % 1);
+    }, intervalDelay);
+    return () => {
+      clearInterval(id);
+    };
+  });
+
+  return <Progress percentComplete={percentComplete} />;
```

Comment:
The default story should take `props: ProgressProps` as a parameter, and spread them here:
```suggestion
  return <Progress percentComplete={percentComplete} {...props} />;
```